### PR TITLE
Added basic export API endpoint

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

**Comment by ErikBjare** [comment]
The commit history became a bit nasty after I merged in the work in the dev/export-import branch (#36)

File: aw_server/api.py [hunk 1]
```diff
@@ -95,6 +104,8 @@ def get_events(self, bucket_id: str, limit: int = -1,
                    start: datetime = None, end: datetime = None) -> List[Event]:
         """Get events from a bucket"""
         logger.debug("Received get request for events in bucket '{}'".format(bucket_id))
+        if limit is None:  # Let limit = None also mean "no limit"
```
[the remark this answers is not in the document]
Comment (ErikBjare) [reply aw_server/api.py]: I just remember someone (I think it was @nikanar, but not sure) suggesting that a limit of `-1` meaning "no limit" was ambiguous and that they prefer `limit=None` for "no limit". I think this is a reasonable change. For now we'd keep `-1` for compatibility though.

[the remark this answers is not in the document]
Comment (nikanar) [reply aw_server/api.py]: Issue was https://github.com/ActivityWatch/aw-client/issues/23. 

`-1` meaning "no limit" is totally fine for me, it's a standard convention. I was surprised by `limit = None` effectively returning a limit of 100, as I would expect `limit = None` to have the same result as `limit = -1`. I think `-1` is less ambiguous.

[the remark this answers is not in the document]
Comment (ErikBjare) [reply aw_server/api.py]: @nikanar Gotcha! Thanks for the input :slightly_smiling_face:

**Comment by ErikBjare** [comment]
@johan-bjareholt I don't know why we haven't merged this, fine with you?

**Pull request merged**